chapters/06-summary.Rmd

```diff
@@ -0,0 +1,25 @@
+# Summary {#summary}
+
+The benchmarking has shown that the new implementation can outperform the de-facto state of the art library 'KDEpy' in terms of runtime and accuracy for large datasets ($n \geq 10^8$).
+
+For smaller datasets it provides accuracy of the same order of magnitude and may be favorable (faster) in cases where the PDF is built once but evaluated repeatedly. 
+
+Both cases are important in high energy physics.
+
+Additionally it has the benefit of allowing any distribution of the loc-scale family that follows the Distribution contract of TensorFlow Probability[@googleTFP] as kernel function. This includes twelve distributions at the moment, namely `Cauchy`, `DoublesidedMaxwell`, `Gumbel`, `Laplace`, `LogLogistic`, `LogNormal`, `Logistic`, `LogitNormal`, `Moyal`, `Normal`, `PoissonLogNormalQuadratureCompound`, `SinhArcsinh`.
+
+The proposed implementation restricts itself to the one-dimensional case as described in section \@ref(introduction-univariate). So far only KDEpy and Statsmodels implement a multidimensional KDE.
+Generalization to higher dimensionel kernel density estimation is feasible, although this would require substantially more work due to some different APIs for multi-dimensional data in TensorFlow. In addition one must ensure that the kernels used are multidimensional themselves, which is not the case for many of the TensorFlow Probability distributions specified above.
```

Comment:
Not sure about the substantial work and the different API though, but I agree that it requires more work as the generalization to ndims is not straightforward for the approximations I would say?

Reply:
Especially since TFP _has_ multidimensional gaussian distributions, right? The most important kernels